dask/dataframe/io/sql.py

```diff
@@ -33,8 +35,9 @@ def read_sql_table(
     ----------
     table : string or sqlalchemy expression
         Select columns from here.
-    uri : string
-        Full sqlalchemy URI for the database connection
+    con : SQLAlchemy connectable or str
+        A database URI could be provided as as str.
```

Comment:
str or SQLAlchemy Engine.
The next comment about sqlite is likely not understood by anyone, including me. No Engine instance is supported with processes.

Reply:
updated type description, removed comment abt SQLite DBAPI

this was copied from Pandas, but I guess we are doing something more than just passing the value along, so customizing our docs makes sense…